CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - feat: Support custom CA roots ([#27062](https://github.com/getsentry/sentry/pull/27062)), see the [docs](https://develop.sentry.dev/self-hosted/custom-ca-roots/) for more details.
 - fix: Fix `curl` image to version 7.77.0
+- upgrade: docker-compose version to 1.29.2
```

Comment:
This is misleading as it refers to the version we test against, not the minimum required version. Cf. #1062.